### DISCUSSION TITLE
I cannot get my first post to upload

### DISCUSSION THREAD

**Discussion by hdaisywd** [MrGreensWorkshop/MrGreen-JekyllTheme#166] [Problems] [answered]
lng_pair: id_first_content
title: 첫번째 게시글

author: Daisy Hong
category: clutter
tags: [clutter]
img: ":post1.jpeg"

date: 2023-01-11 15:17:00 +0900

This is how I made the .markdown file in _posts file! i don't see what's wrong. 
Please tell me if I did something wrong.

**Comment by hdaisywd** [comment, marked as answer]
I found the answer! 
I did not put the exact time and date! 😂😭

**Reply by MrGreensWorkshop** [reply]
I'm glad to know that you fixed the problem.
